services/app-web/src/components/ChangeHistory/ChangeHistoryV2.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { dayjs } from '../../common-code/dateHelpers/dayjs'
+import { SectionHeader } from '../SectionHeader'
+import { Accordion, Link } from '@trussworks/react-uswds'
+import type { AccordionItemProps } from '@trussworks/react-uswds/lib/components/Accordion/Accordion'
+import { UpdateInformation, Contract } from '../../gen/gqlClient'
+import styles from './ChangeHistory.module.scss'
+type ChangeHistoryProps = {
+    contract: Contract
+}
+
+type flatRevisions = UpdateInformation & {
+    kind: 'submit' | 'unlock'
+    revisionVersion: string | undefined
+}
+
+export const ChangeHistoryV2 = ({
+    contract,
+}: ChangeHistoryProps): React.ReactElement => {
+    const flattenedRevisions = (): flatRevisions[] => {
+        const result: flatRevisions[] = []
+        //Reverse revisions to order from earliest to latest revision. This is to correctly set version for each
+        // contract & recontract.
+        const reversedRevisions = [...contract.packageSubmissions].reverse()
```

Comment:
Do we want to put package submissions in time order? I was matching revisions b/c calling contract.packageSubmissions[0] feels like a nice way to always get the most recent submission but could be convinced otherwise

Reply:
@macrael I think we do if we want to keep the order the same as what currently appears (unless I'm missing something?)
<img width="865" alt="Screenshot 2024-04-18 at 2 50 16 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/8bd83b5e-c38e-41fe-aa63-2db25673a5b6">

Reply:
I don't think so. Multiple pages use that `contract.packageSubmissions[0] ` logic already and its working well. This is edge case - I think it's literally just for that integer in the previous submission urls why we reverse in this order -  everywhere else assumes latest first.